docs/DEVELOPMENT.md

```diff
@@ -0,0 +1,84 @@
+# Development Guide
+
+This document provides guidelines and instructions for setting up the development environment and contributing to the project.
+
+## Prerequisites
+
+Ensure you have the following tools installed:
+
+- Golang compiler supported by the Go team (1.22+)
+- Operator SDK version (1.35.x+)
+- Access to OpenShift cluster
+- Quay.io account to store container images
```

Comment:
minor - this list looks like a duplicate of what already present in the `## Getting Started` section, maybe we can create a prerequisites.md file and link it in both . But feel free to leave it as is for now.